lib/oath/views/oath_screen.dart

```diff
@@ -125,6 +131,27 @@ class _UnlockedViewState extends ConsumerState<_UnlockedView> {
     final numCreds = ref.watch(credentialListProvider(widget.devicePath)
         .select((value) => value?.length));
     final hasActions = ref.watch(featureProvider)(features.actions);
+    final qrScanner = ref.watch(qrScannerProvider);
+    final withContext = ref.read(withContextProvider);
+    final credentials = ref.read(credentialsProvider);
```

Comment:
I think these should be moved into the `onFileDropped` function below, to where they are used.

Reply:
I pushed a commit that solves this.